docs/tutorials/basic_commands.md

```diff
@@ -1,23 +1,19 @@
 # Getting Started with Ubuntu Pro Client
 
-The Ubuntu Pro Client (`pro`) provides users with a simple mechanism to
-view, enable, and disable offerings from Canonical on their system. In this tutorial
-we will cover the base `pro` commands that allow a user to successfully manage the offering
-on their machine.
+The Ubuntu Pro Client (`pro`) provides a simple mechanism for viewing, enabling and
+disabling Canonical offerings on your system. In this tutorial we will cover the base
+`pro` commands that will allow you to successfully manage the offerings on your machine.
 
 ## Prerequisites
 
-On this tutorial, you will use [LXD](https://linuxcontainers.org/lxd/) containers.
-To set up LXD on your computer, please follow this [guide](https://linuxcontainers.org/lxd/getting-started-cli/).
-
-```{tip}
-When configuring LXD on your machine, run the `lxd init --minimal`. This should be more
-than enough for you to finish this tutorial
-```
+In this tutorial, you will use [Multipass](https://multipass.run/) virtual machines.
+To install multipass on your computer, please follow this [guide](https://multipass.run/install).
+Note that this page also instructs you how to launch Multipass virtual machines (VM) and interact with
```

Comment:
I actually didn't like the Multipass tutorial at all - I found it confusing and lacking any sort of context. It was just a list of random commands with no explanation of what they are or when to use them. Since you actually tell the users which commands to use in order to complete *this* tutorial, I strongly recommend just telling them the command for installing Multipass here (sudo snap install multipass), and only pointing users to the Multipass tutorial if they're using Windows or MacOS. That command was the only thing I used from the Multipass tutorial, because none of the others are explained.

This may incidentally require also providing a link to snap (as the Multipass tutorial does), or telling them to how to install that - but Ubuntu machines with come with the snap store by default, right? So again, it would just be for non-Ubuntu users

Reply:
+1